tests/Integration/ContentHelperFeatureTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Integration Tests: Base class for all Content Helper feature tests
+ *
+ * @package Parsely\Tests
+ * @since   3.9.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\Tests\ContentHelper;
```

Comment:
Should we also use Snake Case here?

Reply:
I thought about this as well, but currently all our PHP test names do not use snake case. So be convention, I put it like that. We could address this in a refactoring PR.

Reply:
My recommendation: Leave this as is for the moment.